### PR TITLE
Add dependencies support

### PR DESCRIPTION
**Pull request by andresmgot**
This PR adds support for having a file `requirements.txt` next to the handler file to install those requirements for the deployed function.

### CODE REVIEW

File: deploy/kubelessDeploy.js [hunk 1]
```diff
@@ -48,87 +50,118 @@ class KubelessDeploy {
...
-            if (counter === _.keys(this.serverless.service.functions).length) {
-              if (_.isEmpty(errors)) {
-                resolve();
-              } else {
-                reject(
-                  `Found errors while deploying the given functions:\n${errors.join('\n')}`
-                );
-              }
-            }
+        } else {
```
Comment (tompizmor) [inline deploy/kubelessDeploy.js]: Just a comment. I don't know how difficult would be to extend this for other runtimes. For example, I don' know if kubeless support installing dependencies from a package.json file for nodejs function

**Comment by tompizmor** [comment]
LGTM

File: deploy/kubelessDeploy.js (hunk 1 above)
Comment (sebgoa) [inline deploy/kubelessDeploy.js]: yes it does. you can install a `requirements.txt` for python and `package.json` for node.

Comment (andresmgot) [inline deploy/kubelessDeploy.js]: Yes, I prepared the field for separating the language specific files (nodejs is still not supported).

**Pull request merged**